contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-rule.y

```diff
@@ -4226,8 +4222,7 @@ tsql_OptTranName:
 tsql_IsolationLevel:
 			ISOLATION LEVEL tsql_IsolationLevelStr
 				{
-					$$ = list_make1(makeDefElem("transaction_isolation",
-									makeStringConst($3, @3), @1));
+					bbf_set_tran_isolation($3);
```

Comment:
I do not think it is ideal to do the real change as part of parsing. Let us have an internal C function can call set isolation level from there